dot/utils_test.go

```diff
@@ -64,7 +63,7 @@ func TestCreateJSONRawFile(t *testing.T) {
 				bs: &BuildSpec{genesis: NewTestGenesis(t)},
 				fp: filepath.Join(t.TempDir(), "/test.json"),
 			},
-			expectedHash: "23356cdb5d3537d39b735726707216c9e329c7b8a2c8a41b25da0f5f936b3caa",
+			expectedHash: "6453fb6118dab594944a7ed68111fdccc3a68253c9d71ac05f27aa2359507c85",
```

Comment:
Why did this change? 🤔

Reply:
Dan changed the runtime for both `dev` and `gssmr` chains.